weavetest/assert/assert.go

```diff
@@ -7,8 +7,15 @@ import (
 	"github.com/iov-one/weave/errors"
 )
 
+// Asserter is the minimal subset of testing.TB needed to run most assert commands
+type Asserter interface {
+	Helper()
+	Fatal(...interface{})
+	Fatalf(string, ...interface{})
+}
+
 // Nil fails the test if given value is not nil.
-func Nil(t testing.TB, value interface{}) {
+func Nil(t Asserter, value interface{}) {
```

Comment:
Shouldn't the variable be called `a` (not `t`) now that the type is called `Asserter` (not `T`)? :nail_care:

Reply:
Will rename Interface

Reply:
I think most callers will like to see t autocomplete